packages/shortest/README.md

```diff
@@ -73,6 +74,39 @@ shortest('Login to the app using username and password', {
 });
 ```
 
+## Test Dependencies
+You can chain tests to depend on other tests. The dependent tests will only run after their required tests have completed successfully:
+
+```typescript
+import { shortest } from '@antiwork/shortest';
+
+shortest('Login to the application', { 
+  email: 'test@shortest.com', 
+  password: 'password' 
+})
+.expect('User should be logged in successfully');
+
+shortest('Create a new post', { 
+  title: 'Test Post', 
+  content: 'This is a test' 
+})
+.requires('Login to the application')  // This test will run after login
```

Comment:
So something I really like about the library as it exists currently is that it's really elegant and concise, requiring only the `shortest` function call to run a test. IMO adding too many types of chained methods kind of defeats that.

I think it should be possible to have this type of call structure:

```
shortest("test spec")
    .then("dependent test spec 1")
    .then("dependent test spec 2");
``` 
 
What do you think?

Reply:
it will still have that problem if want to run it like:
```typescript
shortest("test spec")

shortest("dependent test spec 2");

shortest("dependent test spec 1")
    .requires("test spec", "dependent test spec 2")
```

There is no equivalent way to do it in that approach.

Reply:
we can do 

```typescript
shortest("test spec")
    .then("dependent test spec 2")
    .then("dependent test spec 1");
```

but i will force me to let "dependent test spec 2" depend on "test spec" even if I don't want to.

Reply:
Instead of using magic strings, could we assign a variable to the test and then have that be the input, versus a string that matches?

E.g

login = shortest("user can login")

after(login).shortest("user can create post")

or

shortest("user can create post").depends_on(login)

(I prefer the first as more readable.)

Reply:
I like the first one too, and there is no constraints when we add multiple dependencies.

after(login, click_settings).shortest("change language to arabic")